### PR TITLE
Private Loans and Totals!

### PR DESCRIPTION
**Pull request by mistergone**
This PR fixes a lot of outstanding issues about adding, removing, and calculating multiple Private Loans. Also, it fixes most of the displayed totals in Part 1.
## Additions
- Added user interface for adding and removing private loans
## Removals
- Removed some code which was redundantly calculating totals which are already calculated in `student-debt-calc`
## Changes
- Update version of student-debt-calc to support multiple private loans
- Updated dispatchers to handle multiple private loans
- Updated DOM and JS to display totals properly
## Testing
- Pull it down, getting it running per the README, and just have a great time!
## Review
- @marteki and @niqjohnson and @higs4281, and maybe @ascott1 if he has some time
## Screenshots

screen shot 2016-01-20 at 3 01 51 pm
screen shot 2016-01-20 at 3 02 32 pm
## Todos
- Fix debt totals
- Fix interest rates (currently not being converted to decimals)
## Checklist
- [x] Changes are limited to a single goal (no scope creep)
- [x] Code can be automatically merged (no conflicts)
- [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
- [x] Passes all existing automated tests
- [x] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)
## Special Thanks

Thanks to @ascott1 and @niqjohnson and @marteki for guiding me through the new stuff!
## Animated GIF

shelob

### CODE REVIEW

File: paying_for_college/templates/worksheet.html [hunk 1]
```diff
@@ -1410,7 +1689,7 @@ <h4 class="aid-form_summary-heading">
                                             $
                                         </span>
                                         <span class="line-item_amount"
-                                        data-financial="totalRepayment"
+                                        data-financial="totalDebt"
```
Comment (marteki) [inline paying_for_college/templates/worksheet.html:1692]: Why this change in attribute name? Especially considering that in the copy of the page, "total debt" refers to the debt from a single offer for one year. The summary box right above this uses the exact phrasing to refer to a totally different number.

It might get confusing.

File: test/functional/settlementAidOfferPage.js [hunk 2]
```diff
@@ -225,7 +225,7 @@ settlementAidOfferPage.prototype = Object.create({}, {
     // TODO: Refactor this here and in the HTML/CSS for multiple private loans?
     privateLoanAmount: {
       get: function() {
-        return element( by.id( 'contrib__private-loan' ) );
+        return element( by.css( '[data-private-loan] [data-private-loan_key="amount"]' ) );
```
Comment (marteki) [inline test/functional/settlementAidOfferPage.js:228]: And yay, updates to the functional test page model! :smile: Can we get the functional test spec updated as well? Right now, none of this work is being verified through actual tests. (As offered before, I can pair with you on getting this done if it makes it easier.)

File: paying_for_college/templates/worksheet.html (hunk 1 above)
Comment (mistergone) [inline paying_for_college/templates/worksheet.html]: All the variable changes are there to reflect the names of variables in `student-debt-calc`.

That said, this is my mistake - this should be a different thing, and I might have to modify student-debt-calc to add this value... :grin:

File: test/functional/settlementAidOfferPage.js (hunk 2 above)
Comment (mistergone) [inline test/functional/settlementAidOfferPage.js:228]: Stay tuned for my next PR! Thrills! Chills! A bunch of functional tests for this stuff!

File: paying_for_college/templates/worksheet.html (hunk 1 above)
Comment (mistergone) [inline paying_for_college/templates/worksheet.html:1692]: All the variable changes are there to reflect the names of variables in `student-debt-calc`.

I get what you're saying, but we're dealing with a million different things called "total" and "debt". We can change the variable name in `student-debt-calc` and trickle it down, but I think matching variable names to copy only works until the copy changes. :smile: We can discuss it further not-in-the-comments-section. :grin:

**Comment by ascott1** [comment]
Changes look good to me! :+1:

**Comment by mistergone** [comment]
Merging! :+1:
